docs/best-practices/code-reviews/readme.md

```diff
@@ -1,129 +1,166 @@
-# Code Reviews
-
-![alt codeQuality](https://imgs.xkcd.com/comics/code_quality.png "Code Quality")
+# Code reviews
+
+![XKCD Code Quality](https://imgs.xkcd.com/comics/code_quality.png)
+[XKCD 'Code Quality', copied under CC BY-NC 2.5](https://xkcd.com/1513/)
```

Comment:
nit: this text is now inline with the image, it needs a newline preceding it

Reply:
done